src/services/organizationService.ts

```diff
@@ -348,3 +349,29 @@ export async function registerOrganization(
     return left(ResponseErrorInternal(genericError));
   }
 }
+
+export async function getOrganizationInstanceFromDelegateEmail(
+  userEmail: string
+): Promise<Either<Error, Option<OrganizationModel>>> {
```

Comment:
not needed for this PR but you can use a TaskEither instead of a promise<either>:
https://github.com/gcanti/fp-ts/blob/master/src/TaskEither.ts

it has a `tryCatch` method and the same semantic as `Either`